### PR TITLE
Allow mutliple ref assignment in textfield to fix autosuggest behaviour

### PR DESCRIPTION
**Pull request by mattsjones**
- Also add route for github pages.

### CODE REVIEW

File: react/fields/Autosuggest/Autosuggest.test.js [hunk 1]
```diff
@@ -76,6 +85,11 @@ describe('Autosuggest', () => {
     expect(suggestionsContainer.className).to.not.contain('suggestionsContainer_withLabel');
   });
 
+  it('should render suggestions list below input field when a label is not supplied', () => {
```
Comment (mattsjones) [inline react/fields/Autosuggest/Autosuggest.test.js]: This is weird, Github is lying. This code is already in master.

File: react/fields/Autosuggest/Autosuggest.test.js [hunk 2]
```diff
@@ -30,7 +32,13 @@ const getAutosuggestProps = (suggestions = []) => ({
 });
 
 describe('Autosuggest', () => {
-  let element, autosuggest, errors, suggestion, suggestionsContainer;
+  let element, autosuggest, errors, suggestions, suggestionsContainer, input;
+
+  const clickSuggestion = suggestion => {
+    Simulate.mouseDown(suggestion);
+    Simulate.mouseUp(suggestion);
+    Simulate.click(suggestion);
```
Comment (mjt01) [inline react/fields/Autosuggest/Autosuggest.test.js]: Did you need to do this? This complexity was only added for `ClearField` interaction, you should be able to simply use `Simulate.click`?

File: react/fields/Autosuggest/Autosuggest.test.js (hunk 1 above)
Comment (mjt01) [inline react/fields/Autosuggest/Autosuggest.test.js]: This test is duplicated now! Check line 83

Comment (mattsjones) [inline react/fields/Autosuggest/Autosuggest.test.js]: Ahh... right

File: react/fields/Autosuggest/Autosuggest.test.js (hunk 2 above)
Comment (mattsjones) [inline react/fields/Autosuggest/Autosuggest.test.js]: I'll double check

**Comment by mjt01** [comment]
:art:

**Pull request merged**